shared/src/models/viewport.ts

```diff
@@ -21,14 +21,20 @@ export class Viewport {
     public readonly type = 'viewport';
 
     /**
-     * top-left position
+     * Position of the viewport.
+     * For positive {@link size}s, this refers to the upper left corner of the viewport, but during resizing, corners may be swapped.
```

Comment:
```suggestion
     * For positive {@link size}s, this refers to the upper left corner of the viewport, but after resizing, corners may be swapped.
```

Same argument as with the simulated regions.